internal/output/diff.go

```diff
@@ -300,3 +300,10 @@ func getSym(d decimal.Decimal) string {
 
 	return ""
 }
+
+func getSymForPercent(d decimal.Decimal) string {
```

Comment:
I would just put this logic in the `formatPercentChange` function. Having a function called `getSym` and `getSymForPercent` might be a bit misleading, because the `getSymForPercent` is really for displaying a symbol assuming the number isn't absolute.